src/blue-magic/blue-magic-caster.cpp

```diff
@@ -136,12 +137,12 @@ static bool cast_blue_teleport_back(PlayerType *player_ptr)
         return false;
     }
 
-    const auto [resisted, m_name] = exe_blue_teleport_back(player_ptr);
-    if (resisted) {
+    const auto m_name = exe_blue_teleport_back(player_ptr);
+    if (!m_name.has_value()) {
         return true;
     }
 
-    msg_format(_("%sを引き戻した。", "You command %s to return."), m_name.data());
+    msg_format(_("%sを引き戻した。", "You command %s to return."), m_name.value().data());
```

Comment:
Using the arrow operator, you can shorten it to m_name->data().

Reply:
That should be done in the forced push.